redbot/cogs/mod/settings.py

```diff
@@ -199,3 +202,19 @@ class ModSettings(MixinMeta):
                     command=f"{ctx.prefix}unban"
                 )
             )
+
+    @modset.command()
+    @commands.guild_only()
+    async def defaultdays(self, ctx: commands.Context, days: int = 0):
+        """Set the default days worth of messages to be deleted when a user is banned.
+        The amount of days has to be between 0 and 7.
+        """
+        guild = ctx.guild
+        if not (0 <= days <= 7):
+            return await ctx.send(_("Invalid days. Must be between 0 and 7."))
```

Comment:
I think this will be clearer if you add "number of" here.
```suggestion
            return await ctx.send(_("Invalid number of days. Must be between 0 and 7."))
```